From f9745d978e6797c4576b52f602325db02dc48403 Mon Sep 17 00:00:00 2001
From: Robert Piasek <dagger@gentoo.org>
Date: Thu, 19 Aug 2010 16:33:59 +0100
Subject: [PATCH] fix iptables-test to work with iptables 1.4.9

Starting with version 1.4.9 iptables dropped xtables_set_revision
function. This patch removes any use of it.
---
 tools/iptables-test.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/tools/iptables-test.c b/tools/iptables-test.c
index 1f3235c..b97c520 100644
--- a/tools/iptables-test.c
+++ b/tools/iptables-test.c
@@ -110,7 +110,7 @@ static struct ipt_entry *build_quota_drop_entry(void)
 		return NULL;
 	m->m->u.match_size = match_size;
 	strcpy(m->m->u.user.name, m->name);
-	xtables_set_revision(m->m->u.user.name, m->revision);
+	m->m->u.user.revision = m->revision;
 	if (m->init != NULL)
 		m->init(m->m);
 
@@ -125,7 +125,7 @@ static struct ipt_entry *build_quota_drop_entry(void)
 	t->t = xtables_calloc(1, target_size);
 	t->t->u.target_size = target_size;
 	strcpy(t->t->u.user.name, "DROP");
-	xtables_set_revision(t->t->u.user.name, t->revision);
+	t->t->u.user.revision = t->revision;
 	if (t->init != NULL)
 		t->init(t->t);
 
-- 
1.7.2

